marketplace.xml

```diff
@@ -13057,4 +13057,39 @@ Uses the MarkLogic Data Movement Java SDK. Works with MarkLogic 9.0 and above on
     </version>
   </versions>
 </market_entry>
+  <market_entry>
+    <id>PentahoGoogleSheetsPlugin</id>
+		<type>Step</type>
+    <name>Pentaho Google Sheets Plugin APIV4</name>
+    <category>
+      <name>Input/Output</name>
+      <parent>
+        <name>PDI Plugins</name>
+      </parent>
+    </category>
+    <description>With these plugins you can read/write from/to Google Spreadsheets. This plugin uses the latest Google API's since others are deprecated March 2020</description>
+    <author>jfmonteil@gmail.com</author>
+    <author_url></author_url>
+    <author_logo>https://avatars0.githubusercontent.com/u/8617022?s=460&v=4</author_logo>
```

Comment:
hi. this is an invalid xml (the & character needs to be replaced with &amp;)

Reply:
oups sorry you mean "&" by "`&amp;`" ?

Reply:
yeah :) github formatted my message. that's exactly it

Reply:
updated the file, many thanks, sorry, first plugin to the marketplace !

Reply:
Hey Pedro, found a little bug, I have produced a 4.1 version, how do we do to proceed for the marketplace ?

Reply:
change the package_url field to point to the new version and I'll merge the pr afterwards.

Reply:
Ok just did it